src/components/ui/post/detail/index.tsx

```
@@ -1,387 +1,249 @@
+    setEditingPostContent,
+    handleEditPost,
+    saveEditPost,
+    cancelEditPost,
+    removeImage,
+    handleImageUpload,
+    removeNewImage,
+    displayDesc,
+  } = usePostEdit(postInfo, async (postId: string) => {
+    const { PostDetailService } = await import("@/services/postDetail");
+    const { useApollo } = await import("@/lib/apolloClient");
+    const client = useApollo();
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)